subsys/bluetooth/host/keys.h

```diff
@@ -46,7 +46,13 @@ struct bt_csrk {
 struct bt_keys {
 	u8_t                    id;
 	bt_addr_le_t		addr;
-	u8_t                    storage_start[0];
+#if defined (CONFIG_BT_KEYS_OVERWRITE_OLDEST) && !defined (CONFIG_BT_KEYS_OVERWRITE_OLDEST_CNTR_IN_SETTINGS)
```

Comment:
You'll need to split this on two lines. Do the splitting after the `&&` and align with the opening `(` of the first line

Reply:
Ah, actually there isn't an opening `(` you might want to add one before the first `defined` so that the grouping is clear